benchmark/src/main/java/ai/timefold/solver/benchmark/impl/report/WebsiteResourceUtils.java

```diff
@@ -5,33 +5,54 @@
 import java.io.InputStream;
 import java.nio.file.Files;
 
+import org.webjars.WebJarAssetLocator;
+
 public class WebsiteResourceUtils {
 
     private static final String RESOURCE_NAMESPACE = "/ai/timefold/solver/benchmark/impl/report/";
 
     public static void copyResourcesTo(File benchmarkReportDirectory) {
-        // Twitter Bootstrap
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/css/bootstrap-responsive.css");
-        // copyResource(benchmarkReportDirectory, "twitterbootstrap/css/bootstrap-responsive.min.css");
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/css/bootstrap.css");
-        // copyResource(benchmarkReportDirectory, "twitterbootstrap/css/bootstrap.min.css");
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/css/prettify.css");
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/img/glyphicons-halflings-white.png");
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/img/glyphicons-halflings.png");
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/js/bootstrap.js");
-        // copyResource(benchmarkReportDirectory, "twitterbootstrap/js/bootstrap.min.js");
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/js/jquery.js");
-        // copyResource(benchmarkReportDirectory, "twitterbootstrap/js/jquery.min.js");
-        copyResource(benchmarkReportDirectory, "twitterbootstrap/js/prettify.js");
-        // Website resources
-        copyResource(benchmarkReportDirectory, "website/css/benchmarkReport.css");
-        copyResource(benchmarkReportDirectory, "website/img/timefold-logo.png");
+        // Extract webjars.
+        File webjarDirectory = new File(benchmarkReportDirectory, "website/webjars/");
+        new WebJarAssetLocator()
```

Comment:
This copies *the entire webjars*, incl. licenses and possibly other stuff we don't need.
As a result, the benchmark report now carries ~20 MB of stuff.
Maybe we want to write code to prune this before we merge?